Dockerfile

```diff
@@ -1,11 +1,20 @@
-FROM golang:1.9
+# Builder image
+FROM golang:1.9 as builder
```

Comment:
Suggest to use `golang:1.9.2` instead.

Reply:
It’s same:
1.9.2, 1.9, 1, latest: 1.9.2-stretch
With 1.9 we take latest 1.9.c